.changeset/eight-hounds-develop.md

```diff
@@ -0,0 +1,10 @@
+---
+'eslint-config-seek': patch
+---
+
+Remove local eslint rule added in [`v11.1.2`]
+
+The local `unsafe-to-chain-command` cypress rule required `eslint-plugin-local-rules` to function, but that plugin doesn't work correctly when yarn hoists it to the top of `node_modules`.
+The application of the rule isn't particularly common, and it's not worth breaking yarn consumers, so removing it for now until either the fork or the upstream `eslint-plugin-cypress` supports it.
```

Comment:
> The application of the rule isn't particularly common

Looks like there is [one consumer](https://github.com/search?q=org:SEEK-Jobs%20unsafe-to-chain-command&type=code) that uses the mentioned rule. Have you considered https://github.com/not-an-aardvark/eslint-plugin-rulesdir as an alternative to `eslint-plugin-local-rules`?

Reply:
That plugin looks like it'll solve the issue since I can resolve the package explicitly.